### PR TITLE
routes for cancelling membership (paid & free)

### PR DESCRIPTION
**Pull request by twrichards**
Added new endpoint...
`/user-attributes/me/cancel-membership`

... which make use of the existing `cancelSubscription` function (used for cancelling regular contributions).

Needed to support Supporter Experience team's new online cancellation flow (online saves OKR).

### CODE REVIEW

**Comment by johnduffell** [comment]
you can't make much simpler PR than that! 👍

**Comment by johnduffell** [comment]
although having said that, contributor+paid-plan+free-plan = everything?  so why not make the one end point to cancel anything?

**Comment by svillafe** [comment]
Following @johnduffell's comment, is it possible to merge both endpoints using [`Member`](https://github.com/guardian/membership-common/blob/91523b8ce30603a3b1072e65456a6d237b78b5bf/src/main/scala/com/gu/memsub/subsv2/Plan.scala#L452)?

File: membership-attribute-service/conf/routes [hunk 1]
```diff
@@ -15,14 +15,17 @@ OPTIONS    /user-attributes/me/membership-update-card       controllers.AccountC
 POST       /user-attributes/me/digitalpack-update-card      controllers.AccountController.digitalPackUpdateCard
 OPTIONS    /user-attributes/me/digitalpack-update-card      controllers.AccountController.digitalPackUpdateCard
 
-POST       /user-attributes/me/paper-update-card      controllers.AccountController.paperUpdateCard
-OPTIONS    /user-attributes/me/paper-update-card      controllers.AccountController.paperUpdateCard
+POST       /user-attributes/me/paper-update-card            controllers.AccountController.paperUpdateCard
+OPTIONS    /user-attributes/me/paper-update-card            controllers.AccountController.paperUpdateCard
```
Comment (svillafe) [inline membership-attribute-service/conf/routes]: I would probably avoid adding removing spaces so the PR keeps simple

Comment (twrichards) [reply membership-attribute-service/conf/routes]: this was just to have everything inline vertically

**Pull request merged**